File: lib/result/Wup.java
package com.teamlab.communication.wireless.lpr9201.result;

/**
 * リザルト Wup
 */
public class Wup extends Result {
	/**
	 * 
	 */
	public static final int RESULT_CODE = 0x86;

	/**
	 * 
	 */
	public static final int DATA_LENGTH_SIZE = 1; // byte
	
	/**
	 * 
	 * @param datas
	 */
	public Wup(int[] datas) {
		super(RESULT_CODE, DATA_LENGTH_SIZE, datas);
	}
}

File: lib2/result/wup.js
/* jslint node: true */
"use strict"

/**
 * リザルト Wup
 */

exports.Wup = Wup;

/**
 * 結果コード
 */
Wup.RESULT_CODE = 0x86;

/**
 * データ長バイトサイズ
 */
Wup.DATA_LENGTH_BYTE_SIZE = 1; // byte

/**
 * @param {Result} result
 */
function Wup(result) {
  this.result = result;
}

/**
 * パース可能か
 * @param {Result} result
 */
Wup.canParse = function(result) {
  return result !== null && result.resultCode === Wup.RESULT_CODE;
}
